source/type/type.cpp

```diff
@@ -650,21 +652,23 @@ VJ* MetaVehicleJourney::impl_create_vj(const std::string& uri,
 }
 
 FrequencyVehicleJourney* MetaVehicleJourney::create_frequency_vj(const std::string& uri,
+                                                                 const std::string& name,
                                                                  const RTLevel level,
                                                                  const ValidityPattern& canceled_vp,
                                                                  Route* route,
                                                                  std::vector<StopTime> sts,
                                                                  nt::PT_Data& pt_data) {
-    return impl_create_vj<FrequencyVehicleJourney>(uri, level, canceled_vp, route, std::move(sts), pt_data);
+    return impl_create_vj<FrequencyVehicleJourney>(uri, name, level, canceled_vp, route, std::move(sts), pt_data);
```

Comment:
Out of curiosity, do you know why we have to `std::move` the stop times ? Would be much nicer to have a `const std::vector<StopTime>& sts` as parameter to avoid confusion and misuse of the function.

Reply:
I guess, same applies to `MetaVehicleJourney::create_discrete_vj` further down.

Reply:
Good question, not a big memory of that, it was done there: https://github.com/CanalTP/navitia/pull/1258
The few I remember was a perf issue/concern.

I think the move shouldn't be changed in that PR, although a rename/comment may be added? Suggestions welcome!